source/com.microsoft.tfs.client.eclipse.ui.egit/src/com/microsoft/tfs/client/eclipse/ui/egit/importwizard/GitImportWizardSelectFoldersPage.java

```diff
@@ -134,19 +138,50 @@ private void createRepositoriesTreeControl(final Composite container) {
         GridDataBuilder.newInstance().fill().grab().hHint(150).applyTo(treeControl);
         treeControl.setHeaderVisible(false);
 
-        folderColumn = new TreeColumn(treeControl, SWT.NONE);
-        folderColumn.setText(Messages.getString("GitImportWizardSelectFoldersPage.ProjectColumnName")); //$NON-NLS-1$
-        folderColumn.setWidth(300);
+        folderColumn = new TreeColumn(treeControl, SWT.CENTER);
+        folderColumn.setText(Messages.getString("GitImportWizardSelectFoldersPage.FoldersColumnName")); //$NON-NLS-1$
 
         final TreeViewerColumn nameColumnViewer = new TreeViewerColumn(treeViewer, folderColumn);
         nameColumnViewer.setLabelProvider(new FolderNameLabelProvider());
 
-        treeControl.setHeaderVisible(false);
-
         treeViewer.setUseHashlookup(true);
         treeViewer.setContentProvider(new FolderTreeContentProvider());
         treeViewer.setAutoExpandLevel(2);
 
+        container.addControlListener(new ControlAdapter() {
```

Comment:
I understand why you are doing this, but it seems bad for us to be handling the resize event. SWT should do these kinds of things for us.